.github/workflows/reeek.yml

```diff
@@ -16,10 +16,10 @@ jobs:
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 3.3
+          ruby-version: 3.1.3
 
       - name: Install dependencies
         run: gem install reek
 
       - name: Run Reek
```

Comment:
Why are you editing this file?

Reply:
Because the GitHub Action did not advance in the installation of the dependencies and always gave us an error.

Reply:
check uwu!!